xmm-pysas-intro-short.ipynb

```diff
@@ -0,0 +1,212 @@
+   "id": "9d902e6f-b877-4cb4-8c5c-2e7e78effb34",
+   "metadata": {},
+   "source": [
+    "If you want more information on the function `odfcompile` run the cell below or see the long introduction tutorial."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b14a91f5-08cb-488c-b3a3-b2fa10e72bf1",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "print(odf.odfcompile.__doc__)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4f0945a2-8f6d-49ef-90f4-6805670e1a47",
+   "metadata": {},
+   "source": [
+    "To run SAS tasks, especially ones not written in Python, you will need to import a wrapper from pySAS. SAS tasks should be run from the work directory."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0ee9af23-785d-4350-831e-f12707e2bc3c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from pysas.wrapper import Wrapper as w\n",
+    "os.chdir(odf.work_dir)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a6550002-2b18-4bcb-9f90-277fe42722f1",
+   "metadata": {},
+   "source": [
+    "All SAS tasks are called by inputing the name of the task into the wrapper along with any input arguments."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d48b9624-e2bd-4455-a62b-e3054f80b01e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "inargs = []\n",
+    "w('epproc', inargs).run()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9a7ba5f1-a307-424c-94fe-7ac494cdca36",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w('emproc', []).run()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cb9ccdd1-9409-46c4-97f0-f43d7cc9eb6a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w('rgsproc', []).run()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "29ca78c2-641e-463e-b740-f02fbd22ef40",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w('omichain', []).run()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "72e48655-eab6-440f-83b4-46e4154a56fc",
+   "metadata": {},
+   "source": [
+    "You can list all options available to any SAS task with option `'--help'` (or `'-h'`),"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7ad05c30-1048-4304-b382-b926bed7efe2",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "w('epproc', ['-h']).run()"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.11.5"
+  }
+ },
```

Comment:
Please add a link to the xmm GOF help desk, so users know here to find help if they need it.